core/primitives/src/state_part.rs

```diff
@@ -1,3 +1,15 @@
+use std::fmt;
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_primitives_core::types::ProtocolVersion;
+use near_primitives_core::version::ProtocolFeature;
+use near_schema_checker_lib::ProtocolSchema;
+use strum::AsRefStr;
+
+use crate::state::PartialState;
+
+const STATE_PARTS_COMPRESSION_LEVEL: i32 = 3;
```

Comment:
That works well, but the exact value is to be determined, and we could make it as part of config (that could be different on each peer and still work). Not in this PR.